schnorr_fun/src/message.rs

```diff
@@ -37,6 +37,11 @@ impl<'a, 'b, S: Secrecy> Message<'a, S> {
         }
     }
 
+    /// Check if the message is empty with zero length
+    pub fn is_empty(&self) -> bool {
```

Comment:
Did clippy make you do this?

Reply:
Yeah, something about implementing len but not empty